definitions/divorce/json/AuthorisationCaseState/AuthorisationCaseState.json

```diff
@@ -2308,5 +2308,103 @@
     "CaseStateID": "awaitingServicePayment",
     "UserRole": "caseworker-divorce-bulkscan",
     "CRUD": "R"
+  },
+  {
```

Comment:
ditto. these should only exist in non-prod

Reply:
but why if they would have to be duplicated in prod and nonprod? So they will go to prod xlsx anyway.